packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -332,7 +335,18 @@ class Frame extends React.Component {
       event.preventDefault();
       this.openDropdown(event);
     } else if (keyCode === KeyCode.KEY_BACK_SPACE && !this.state.searchValue && value.length > 0) {
-      this.props.onDeselect(value[value.length - 1]);
+      const unselectedTxt = intl.formatMessage({ id: 'Terra.form.select.unselected' });
+      const lastOptionValue = value[value.length - 1];
+      const lastOption = React.Children.toArray(children).find(child => child.props.value === lastOptionValue);
```

Comment:
I wasn't able to test this on the deployed examples because the values and displays are identical. But I think we may need to use the findByValue utility to search for the appropriate child, specifically for the opt groups. This will do a flat search through the opt group but it won't search through the options of each group.

Reply:
Ah, good catch. Forgot about the OptGroups. Will fix that